aws/resource_aws_dms_endpoint.go

```diff
@@ -217,6 +218,41 @@ func resourceAwsDmsEndpoint() *schema.Resource {
 					},
 				},
 			},
+			"elasticsearch_settings": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					if old == "1" && new == "0" {
+						return true
+					}
+					return false
+				},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"service_access_role_arn": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"endpoint_uri": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"error_retry_duration": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Default:  "10",
```

Comment:
As per https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Target.Elasticsearch.html the default should be 300